docs/source/install/kernel_install.rst

```diff
@@ -53,3 +53,33 @@ For example, using conda environments:
 The ``--name`` value is used by Jupyter internally. These commands will overwrite
 any existing kernel with the same name. ``--display-name`` is what you see in
 the notebook menus.
+
+Using virtualenv you can add your new environments by linking to specific instance of your Python:
+
+.. sourcecode:: bash
+
+    python -m ipykernel install --prefix=/path/to/new_env --name python_new_env
```

Comment:
This is a complicated case and could use some explicit notes about which things point to which Python. Some clarification:

> Using virtualenv or conda envs, you can make your IPython kernel in one env available to Jupyter in a different env. To do so, run `ipykernel install` from the kernel's env, with `--prefix` pointing to the Jupyter env:

    /path/to/kernel/env/bin/python -m ipykernel install --prefix=/path/to/jupyter/env --name 'python-my-env'